tests/model_server_test.py

```diff
@@ -43,8 +45,35 @@ def get_models():
     response = json.loads(response.text)
     return response
 
+def test_attr_has_value():
+    attrs = dict()
+    non_numerical_test_cases = {
+        "": False,
+        None: False,
+        "some": True
+    }
+    numerical_test_cases = {
+        "": False,
+        None: False,
+        "0": False,
+        0: False,
+        "-1": False,
+        -1: False,
+        "1": True,
+        1: True
+    }
+    for tc, value in non_numerical_test_cases.items():
+        attrs[NodeAttribute.PROCESSOR] = tc
+        assert attr_has_value(attrs, NodeAttribute.PROCESSOR) == value
+    for tc, value in numerical_test_cases.items():
+        attrs[NodeAttribute.CORES] = tc
+        assert attr_has_value(attrs, NodeAttribute.CORES) == value
+
+def run_model_server_unit_tests():
+    test_attr_has_value()
 
 if __name__ == "__main__":
+    run_model_server_unit_tests()
```

Comment:
could we move all unit-tests to a `test_<component>.py` file that pytest can run?